pure_pursuit_wrapper/package.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>pure_pursuit_wrapper</name>
+  <version>0.0.0</version>
```

Comment:
make version 0.0.1

Reply:
Also add the license comment to you xml files

Reply:
fix